### ISSUE TITLE
Add A Static/manually set Monitor

### ISSUE THREAD

**Issue by 14wkinnersley** [louislam/uptime-kuma#475]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Monitor

### 🔖 Feature description

There is not option to add a static monitor that can be set up or down manually.

Sometimes you can't monitor several services by the currently available monitors in Uptime Kuma, but for having my users also notified that a service is down, I want to create a monitor which can be turned "offline" manually

### ✔️ Solution

Requesting the ability to add a static monitor that is set either up or down manually.

This is useful to add items like infrastructure items (Cooling/Power) that can be set either up or down depending on what is inputted. This would be great to add to the new public status page.

**Comment by github-479** [comment]
So a way to handle this is just adding a keyword monitor in Kuma monitoring a static HTTP page. 
Like HTTP://myserver.com/hvac.html then just put a word on this page. 

You'd be updating the HTML page manually instead of handling it in Kuma.

**Comment by 14wkinnersley** [comment]
> So a way to handle this is just adding a keyword monitor in Kuma monitoring a static HTTP page. 
> 
> Like HTTP://myserver.com/hvac.html then just put a word on this page. 
> 
> 
> 
> You'd be updating the HTML page manually instead of handling it in Kuma. 

This is an option that I currently have set up now. Its not a great quality of life, but its something. Would love to see this feature implemented in Kuma.

**Comment by Anaphylaxis** [comment]
Yes, would love a "pull" feature where services can be customized to send to a custom webhook in UT-K that's automatically created per monitor.

**Comment by iTzEthqnHD** [comment]
This is the only reason I don't use uptime kuma at the moment. I am in need of static monitors that are manually controlled.

**Comment by CommanderStorm** [comment]
https://github.com/louislam/uptime-kuma/issues/2715#issuecomment-1416648708 and https://github.com/louislam/uptime-kuma/issues/2715#issuecomment-1416889427 have proposed workarounds around this current limtation

**Comment by digitalfiz** [comment]
A potential implementation suggestion would be to allow monitors to add buttons to the page when you're viewing a monitor and its details. Then, a manual monitor can be added that adds buttons for up and down and also any other monitors that may have other custom things they may want to do. I know this adds a good amount of scope creep, but it would be a new feature that could be helpful in the future for other requests.

![Screenshot 2024-08-10 at 5 04 15 PM](https://github.com/user-attachments/assets/44340bfe-0283-4801-b0e8-fbd7694ced70)

**Comment by cascha42** [comment]
Thank you for implementing manual monitoring in version 2.0.
Is it possible to toggle the status of a manual monitor via a GET request?
My external device can send a GET request when an error occurs and another one when the error is resolved.

**Comment by CommanderStorm** [comment]
Please use the push monitor for this instead.

For the manual monitor, this is done => closing as resolved
